crates/sncast/tests/data/scripts/old_sncast_std/contracts/src/lib.cairo

```diff
@@ -0,0 +1 @@
+fn some_func() {}
```

Comment:
This is not imported in your script, and thus not used anywhere. I suppose we could get rid of this whole (contract) directory, as it is non essential to the test.

Reply:
Done